aptos-move/framework/aptos-framework/sources/configs/version.spec.move

```diff
@@ -37,7 +37,7 @@ spec aptos_framework::version {
         pragma verify_duration_estimate = 120;
         include transaction_fee::RequiresCollectedFeesPerValueLeqBlockAptosSupply;
         include staking_config::StakingRewardsConfigRequirement;
-        requires chain_status::is_operating();
+        requires chain_status::is_genesis();
```

Comment:
(1) first of all, `set_version` doesn't seem to be a genesis-only function.

(2) `set_version` seems to be deprecated by `set_for_next_epoch()`.